### PR TITLE
[#3351] Partner info in family component

### LINKED ISSUE #3351: Retrieve Partner-information from HC and StUF-BG when user is logged in with DigiD
Reference: Utr 76

At minimum we need firstname, last name, bsn, birthdate... but maybe we need to retrieve what is needed by prefill? We also can combine this with co-sign to determine who needs to co-sign?

Needs some refinement.

### PR DESCRIPTION
**Pull request by SilviaAmAm**
Fixes #3351

### CODE REVIEW

File: src/openforms/formio/components/np_family_members/tests/responses/stuf_bg_family_members.xml
Comment (SilviaAmAm) [inline src/openforms/formio/components/np_family_members/tests/responses/stuf_bg_family_members.xml:1]: Temporary until we have an actual test response

File: docs/manual/forms/form_fields.rst [hunk 1]
```diff
@@ -230,6 +230,19 @@ Basis
      bijvoorbeeld ``-`` gebruiken. Als weergave kunt u bijvoorbeeld ``-------`` of
      ``- geen keuze -`` gebruiken.
 
+Gezinsleden
+===========
+
+De *Gezinsledenscomponent* toont de namen van alle gezinsleden (partners en/of kinderen) als selectievakjes.
+De waarden die naar de registratie backend worden gestuurd zijn de BSNs.
+
+De gegevens voor de gezinsleden worden opgehaald van Haal Centraal of StUF-BG. Dit kan in de Admin worden geconfigureerd
```
Comment (sergei-maertens) [inline docs/manual/forms/form_fields.rst:239]:
```suggestion
De gegevens voor de gezinsleden worden opgehaald van Haal Centraal BRP Personen Bevragen of StUF-BG. Dit kan in de Admin worden geconfigureerd
```

File: src/openforms/formio/components/custom.py [hunk 2]
```diff
@@ -162,7 +169,11 @@ def mutate_config_dynamically(
             # TODO: this should eventually be replaced with logic rules/variables that
             # retrieve data from an "arbitrary source", which will cause the data to
             # become available in the ``data`` argument instead.
-            child_choices = handler(bsn)
+            child_choices = handler(
+                bsn,
+                include_children=component["includeChildren"],
+                include_partners=component["includePartners"],
```
Comment (sergei-maertens) [inline src/openforms/formio/components/custom.py:175]: perhaps `.get` is more appropriate, in the event that old forms are imported that didn't go through the data migration? Or we need to account for it during the import.

**Review by sergei-maertens — APPROVED** [review]

File: src/openforms/formio/components/custom.py (hunk 2 above)
Comment (SilviaAmAm) [reply src/openforms/formio/components/custom.py:175]: `.get` with the defaults is better I think!

**Pull request merged**